src/RootPath.js

```diff
@@ -0,0 +1,2 @@
+const RootPath = '/';
```

Comment:
rename this file constants.js OR routePaths.js because u will be adding all the paths in this file only.